modules/oe_content_featured_media_field/src/Plugin/Field/FieldType/FeaturedMediaFieldItem.php

```diff
@@ -46,18 +43,6 @@ public static function defaultStorageSettings() {
     ] + parent::defaultStorageSettings();
   }
 
-  /**
-   * {@inheritdoc}
-   */
-  public function storageSettingsForm(array &$form, FormStateInterface $form_state, $has_data) {
-    $element = parent::storageSettingsForm($form, $form_state, $has_data);
-    // Don't allow any other entity type than media to be referenced.
-    $element['target_type']['#default_value'] = 'media';
-    $element['target_type']['#disabled'] = TRUE;
-
-    return $element;
-  }
-
```

Comment:
I'm sorry if I wasn't clear. The storageSettingsForm method it is needed because without it we can change the reference type:
<img width="389" alt="Screen Shot 2020-05-18 at 2 37 00 PM" src="https://user-images.githubusercontent.com/22908988/82213867-260d7300-9915-11ea-8d68-b7678a666218.png">
but the function has to return an empty array as mentioned before, in order to be able to choose the cardinality:
<img width="401" alt="Screen Shot 2020-05-18 at 2 39 33 PM" src="https://user-images.githubusercontent.com/22908988/82214036-708eef80-9915-11ea-8ec7-2155ff1af147.png">

do smth like this:
```/**
   * {@inheritdoc}
   */
  public function storageSettingsForm(array &$form, FormStateInterface $form_state, $has_data) {
    return [];
  }```

Reply:
In any case, we need to decide if we reuse the storage under config/install or not to be able to have different settings for each entity.

Reply:
We can see that later, for now it's there what is in the ticket.